### ISSUE TITLE
Input resolution

### ISSUE THREAD

**Issue by EthanAbitbol3** [PaddlePaddle/PaddleSeg#3439]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

Hello and thank you for your work!
I had a question about the input resolution of an algorithm. I am currently using ade20k which has an image size that varies but for example on segformer it is mentioned that the input size is 512x512, how is this done?

thanks again !

**Comment by Asthestarsfalll** [comment]
By using image interpolation algorithms, which are available in most Python image libraries under the name "resize." Some deep learning frameworks may refer to these algorithms as "interpolate."

**Comment by EthanAbitbol3** [comment]
Thank you for your help and your time ! 

for example in this configuration file : 

_base_: '../_base_/cityscapes_1024x1024.yml'

batch_size: 2
iters: 160000

model:
  type: SegFormer
  backbone:
    type: MixVisionTransformer_B0
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/backbone/mix_vision_transformer_b0.tar.gz
  embedding_dim: 256
  num_classes: 19

optimizer:
  _inherited_: False
  type: AdamW
  beta1: 0.9
  beta2: 0.999
  weight_decay: 0.01

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.00006
  power: 1

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

test_config:
    is_slide: True
    crop_size: [1024, 1024]
    stride: [768, 768]


Does the crop size specified in the `test_config` refer to the input resolution?

**Comment by Asthestarsfalll** [comment]
> Does the crop size specified in the `test_config` refer to the input resolution?

Yes, the input size is [1024, 1024].
By the way, crop is another way to adjust the size of images. It does not alter the scale of image, but crop some part of images
